uber/site_sections/devtools.py

```diff
@@ -19,6 +19,85 @@ def run_git_cmd(cmd):
     return run_shell_cmd(git + " " + cmd, working_dir=uber_base_dir)
 
 
+def generate_test_email_models():
+    test_models = dict()
```

Comment:
This is a static dict, so it seems odd to re-run this function every time we try to look at emails. If this was instead generated on start-up and made a little more globally accessible (maybe put it in the Emails class?), then plugins could easily append to this dict and we would no longer have to do all that try/except stuff.